app/bootstrap.php

```diff
@@ -118,7 +118,7 @@
 
     // Create the 'Bolt application'
     $appClass = Application::class;
-    if ($config['application'] !== null && is_a($config['application'], Silex\Application::class, true)) {
+    if ($config['application'] !== null && is_subclass_of($config['application'], Silex\Application::class)) {
         $appClass = $config['application'];
```

Comment:
I think if it gets to this block it's already not a concrete `Silex\Application` if you look up to line 95 then it returns.

I guess there's an argument for allowing `Silex\Application` as a string but I'm not sure how that would work out in terms of interface.